docs/userguide/datafiles.rst

```diff
@@ -35,16 +37,34 @@ For example, if the package tree looks like this::
             ├── data1.txt
             └── data2.txt
 
-and you supply this configuration:
+When **at least one** of the following conditions are met:
+
+1. These files are included via the :ref:`MANIFEST.in <Using MANIFEST.in>` file,
+   like so::
+
+        include src/mypkg/*.txt
+        include src/mypkg/*.rst
+
+2. They are being tracked by a version control system such as Git, Mercurial
```

Comment:
I guess "version control system" might be more appropriate in this context than ["revision control system"](https://en.wikipedia.org/wiki/Revision_Control_System):

> Revision Control System (RCS) is an early implementation of a [version control system](https://en.wikipedia.org/wiki/Version_control_system) (VCS).

From [the Wikipedia of Git](https://en.wikipedia.org/wiki/Git):
> Git ([/ɡɪt/](https://en.wikipedia.org/wiki/Help:IPA/English))[[8]](https://en.wikipedia.org/wiki/Git#cite_note-:0-9) is a [distributed version control](https://en.wikipedia.org/wiki/Distributed_version_control) [system](https://en.wikipedia.org/wiki/Software_system)[[9]](https://en.wikipedia.org/wiki/Git#cite_note-FOOTNOTEChaconStraub201429%E2%80%9331-10) that tracks versions of [files](https://en.wikipedia.org/wiki/Computer_file).

And [SVN](https://en.wikipedia.org/wiki/Apache_Subversion):

> Apache Subversion (often abbreviated SVN, after its command name svn) is a [version control](https://en.wikipedia.org/wiki/Version_control) system distributed as [open source](https://en.wikipedia.org/wiki/Open-source_software) under the [Apache License](https://en.wikipedia.org/wiki/Apache_License).[[2]](https://en.wikipedia.org/wiki/Apache_Subversion#cite_note-3)

Reply:
I am happy with both.

I believe that the entry in Wikipedia that you cited is referring to a specific "object" in the "category" of Revision Control Systems, also called "Revision Control System" :P, but not the category itself.

For our purposes "Revision Control System" or "Revision Control Software" are synonyms with "Version Control System". Based in your comment, Mercurial seems to consider the same.

It would be interesting however to keep the usage consistent and use the same term across the whole documentation. So if we are changing this paragraph, we should also change elsewhere. Alternatively we can also add a `(also known as Version Control System)` note somewhere...

Reply:
Okay, I would revert this change because these two are to some extent interchangeable, and is not the focus of this PR (plus I know little about terminology used elsewhere across the docs).